Game/src/main/MainApp.java

```diff
@@ -3,6 +3,7 @@
 import loaders.LoaderSettings;
 import loaders.SceneLoader;
 import loaders.SceneLoaderFactory;
+import mini.input.*;
```

Comment:
Nu er det her gjort i MainApp, så det betyder meget mindre, men at bruge * import er dårlig stil. Så for så vidt muligt, brug single imports.